airbyte-integrations/connectors/source-outreach/source_outreach/schemas/sequence_steps.json

```diff
@@ -81,7 +81,7 @@
     "creator": {
       "type": ["null", "array"],
       "items": {
-        "type": "string"
+        "type": ["string", "integer"]
```

Comment:
let's flag this in the breaking change. I think this is important information because it means the field type in the destination and in downstream systems need to support both string and integer

Reply:
Done!